plugin/goto_diagnostic.py

```diff
@@ -0,0 +1,206 @@
+from .core.diagnostics_manager import ParsedUri, is_severity_included
+from .core.protocol import Diagnostic, DiagnosticSeverity, DocumentUri, Location, LocationLink, Position
+from .core.registry import windows
+from .core.sessions import Session
+from .core.settings import userprefs
+from .core.typing import Iterable, Iterator, List, Optional, Tuple, Union
+from .core.url import parse_uri, unparse_uri
+from .core.views import MissingUriError
+from .core.views import diagnostic_severity, format_diagnostic_source_and_code, format_severity, uri_from_view
+from .locationpicker import EnhancedLocationPicker, OnModifierKeysAcitonMap
+from collections import Counter, OrderedDict
+from pathlib import Path
```

Comment:
`pathlib` is not available in the 3.3 runtime that this package runs within. I think that's the reason for tests failing.

Though I believe there is a PC dependency available and we have it installed through another package so it doesn't fail locally.

We could either add `pathlib` to `dependencies.json` or not use `Path`.